data-prepper-plugins/prometheus-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/configuration/AuthTypeOptions.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum AuthTypeOptions {
+    HTTP_BASIC("http_basic"),
```

Comment:
I think for consistency with other values, these should be `http-basic` and `bearer-token`.

Reply:
Thanks for the review @dlvenable , Fixed